zerver/lib/send_email.py

```diff
@@ -306,6 +307,7 @@ def initialize_connection(connection: Optional[BaseEmailBackend] = None) -> Base
 
     if connection.open():
         # If it's a new connection, no need to no-op to check connectivity
+        connection.END_MAX_CONNECTION_MINUTES = int(time.time())
         return connection
```

Comment:
`END_MAX_CONNECTION_MINUTES` is not a good name heree -- this value is the last time the connection was opened, not the time when it ends, or the max of anything, nor does it store minutes.